dartagnan/pom.xml

```diff
@@ -653,6 +653,7 @@
   </dependencies>
 
   <build>
+    <finalName>${project.artifactId}</finalName>
```

Comment:
What's the use of this? Why didn't we need it before?

Reply:
By default the name of the generated jar is `${project.artifactId}-${version}` thus I override the file name to remove the version